lib/presentation/screen/journal_edit_screen.dart

```diff
@@ -7,29 +8,35 @@ import 'package:star_book/theme/styling/theme_color_style.dart';
 
 class JournalEditScreen extends StatelessWidget {
   final String date;
-  final Color moodColor;
+  // final Color moodColor;
   final String mood;
   final String titleDescription;
   final String noteDescription;
 
   const JournalEditScreen({
     Key? key,
     required this.date,
-    required this.moodColor,
+    // required this.moodColor,
     required this.mood,
     required this.titleDescription,
     required this.noteDescription,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    // final double screenWidth = MediaQuery.of(context).size.width;
     final double screenHeight = MediaQuery.of(context).size.height;
     return Scaffold(
       appBar: PrimaryAppBar(
         leading: PrimaryAppBarItem(
           icon: Icons.arrow_back_ios_new_outlined,
           label: 'Back',
+          onTap: () => context.goNamed('JournalDetailScreen', params: {
+            'detailDate': '05 September 2022',
+            // 'detailMoodColor': ,
+            'detailMood': 'Productive',
+            'detailTitleDescription': 'Feeling Productive',
+            'detailNoteDescription': 'Feeling Productive Feeling Productive',
+          }),
```

Comment:
detail screen will only take journal id as parameter. and bloc will resolve the journal based on given id.

Reply:
Okay